vformer/encoder/pyramid.py

```diff
@@ -65,6 +66,25 @@ def __init__(
         )
 
     def forward(self, x, **kwargs):
+        """
+
+        Parameters
+        ----------
+            x: torch.Tensor
+                Input tensor
+        Kwargs
```

Comment:
You don't need to have a separate section for **kwargs. Check [this](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html#:~:text=length%20argument%20list.-,**kwargs,-Arbitrary%20keyword%20arguments)